qiita_ware/processing_pipeline.py

```diff
@@ -83,6 +83,51 @@ def _get_qiime_minimal_mapping(prep_template, out_dir):
     return output_fps
 
 
+def _get_sample_names_by_run_prefix(prep_template):
+    """Generates a dictionary of run_prefix and sample names
+
+    Parameters
+    ----------
+    prep_template : PrepTemplate
+        The prep template from which we need to generate the minimal mapping
+
+    Returns
+    -------
+    dict
+        The paths to the qiime minimal mapping files
+
+    Raises
+    ------
+    ValueError
+        If there is more than 1 sample per run_prefix
+    """
+    from collections import defaultdict
+    import pandas as pd
+
+    # The prep templates has a QIIME mapping file, get it
+    qiime_map = pd.read_csv(prep_template.qiime_map_fp, sep='\t',
```

Comment:
We have identified a potential issue parsing the QIIME mapping files in this way. Can you use `qiime_map = load_template_to_dataframe(fp, index='#SampleID')`? It will solve that problem.

I know this is inconsistent with the other functions in this file, but those instances also need to be changed, so I prefer to not add more potential sources of errors.